### ISSUE TITLE
once iframe2 is merge into master, implement autodeploy of static iframe assets

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#748]
we need to autodeploy the static assets from `iodide.app`. @mdboom i'll give you permissions to change the DNS settings however you need.

blocked by #747

**Comment by mdboom** [comment]
This is deploying just fine to iodide-project/master-eval.  Setting up the redirect from `extremely-alpha.iodide.app` to this is hitting an HTTPS-related snag.  Working on it, and keeping this issue open...

**Comment by mdboom** [comment]
This appears to be working now, though haven't tested end-to-end since we don't have the server and main frame assets deployed yet.
